db/migrate/20190606132429_add_startdate_to_lesson.rb

```diff
@@ -0,0 +1,5 @@
+class AddStartdateToLesson < ActiveRecord::Migration[5.2]
```

Comment:
where does this migration file come from? we dont need it as it is in the migration file of migration_lessons